test/test_ssl/test_dhparam.yml

```diff
@@ -0,0 +1,16 @@
+web5:
+  image: web
+  expose:
+    - "85"
+  environment:
+    WEB_PORTS: "85"
+    VIRTUAL_HOST: "web5.nginx-proxy.tld"
+    HTTPS_METHOD: nohttp
```

Comment:
Good point, no need for it here - it was an artifact of me copying another test :)